pwa-devdocs/src/tutorials/create-taglist-component/index.md

```diff
@@ -0,0 +1,454 @@
+
+The general steps provided by this tutorial can apply to different kinds of setup, but
+the specific details assumes that you followed the steps in the [Venia storefront setup][] topic to set up a storefront for development.
+
+## Step 1. Create component directory and files
+
+Inside your project's directory for components, create a new directory called **TagList**.
+In the `pwa-studio` project, this directory is `packages/venia-ui/lib/components`.
+
+Inside the **TagList** directory, create empty files with the following names:
+
+-   `index.js`
+-   `tag.css`
+-   `tag.js`
+-   `tagList.css`
+-   `tagList.js`
+
+{: .bs-callout .bs-callout-info}
+These names follow the same [naming conventions][] used in the `pwa-studio` project.
+
+## Step 2. Expose the module's public API
+
+Open the component's `index.js` file and add the following content:
+
+```js
+export { default } from './tagList';
+```
+
+Exporting modules from your component's `index.js` file is a commond standard in a React project.
+It allows a component to import this component using its directory name.
+This is demonstrated in a later step.
+
+## Step 3. Define the `Tag` module
+
+Open the `tag.js` file and add the following content:
+
+```jsx
+import React from 'react';
+
+// Use the prop-types module for type checking
+import { string } from 'prop-types';
+
+// This is a module responsible for rendering a single tag
+const Tag = props => {
+
+    // Destructure the props object into variables
+    const { value } = props;
+
+    // Return the tag string inside a list item element
+    return (<li>{value}</li>);
+}
+
+// Define the props this component accepts
+Tag.propTypes = {
+    value: string
+}
+
+// Make this function the default exported module for this file
+export default Tag;
+```
+
+This file defines the Tag component.
+It is responible for rendering a single tag in the tag list.
+
+## Step 4. Define the `TagList` module
+
+Open the `tagList.js` file and add the following content:
+
+```jsx
+import React from 'react';
+
+import { arrayOf, string } from 'prop-types';
+
+// Import the previously defined Tag component
+import Tag from './tag';
+
+// This is the main tag list module
+const TagList = props => {
+
+    // Destructure the props object into variables
+    const { tagArray } = props;
+
+    // Convert the array of tag strings into a list of Tag components
+    const tagList = tagArray.map(value => {
+        return <Tag key={value} value={value} />;
+    })
+
+    // Returns the list of Tag components inside a div container
+    return (<ul>{tagList}</ul>);
+}
+
+TagList.propTypes = {
+    // tagArray is expected to be an array of strings
+    tagArray: arrayOf(string)
+}
+
+export default TagList;
+```
+
+This file defines the main TagList component exported by the `index.js` file.
+It accepts an array of strings and returns a `div` container with a list of tags.
+
+## Step 5. Add component to the product details page
```

Comment:
I might move the current Step 2 (show them how to export it) to right before this step (importing it).